sechub-commons-model/src/main/java/com/mercedesbenz/sechub/commons/model/SecHubConfigurationModelValidator.java

```diff
@@ -552,6 +554,56 @@ private void validateNameUniqueAndNotNull(InternalValidationContext context, Col
 
     }
 
+    private void validateRemoteDataSection(InternalValidationContext context, Collection<? extends SecHubDataConfigurationObject> configurationObjects) {
+
+        SecHubConfigurationModelValidationResult result = context.result;
+
+        for (SecHubDataConfigurationObject configurationObject : configurationObjects) {
+            Optional<SecHubRemoteDataConfiguration> optRemoteData = configurationObject.getRemote();
+
+            if (optRemoteData.isEmpty()) {
+                // no remote data is configured
+                return;
+            }
+
+            String uniqueName = configurationObject.getUniqueName();
+
+            if (configurationObjects.size() > 1) {
+                // remote data is configured with multiple configurations (filesystem or second
```

Comment:
I find this misleading.

Not the remote data is configured with multiple configurations - but we have multiple sources or binaries (configuration objects) and at least one is a remote configuration.

Means if one configurationObject (or "sourceOrBinary") is a remote definition and we have multiple "sourceOrBinary" objects its currently a failure

It is currently not clear if we allow:
a) if one remote defined, no filesystem may be defined
b) if one remote defined, not any other remote maybe defined 
  b1) everywhere for any other source / binary (but handled separatley) (as implemented currently)
  b2) everywhere for any other source + binary (means not two calls but only one call necessary)
c) does the unique id matter in such cases?